db/migrate/20160513160206_initial_migration.rb

```diff
@@ -0,0 +1,34 @@
+class InitialMigration < ActiveRecord::Migration
+  def change
+    create_table :links do |t|
+      t.string :uri
+      t.integer :upvote_count
+      t.integer :downvote_count
+
+      t.timestamps null: false
+    end
+
+    create_table :tags do |t|
+      t.string :name
```

Comment:
If this should be indexed, should the `uri` column of the `links` table be as well? What's a good rule for when/when not to index a column?